Livecheckables/nomad.rb

```diff
@@ -1,6 +1,5 @@
 class Nomad
   livecheck do
-    url "https://www.nomadproject.io/downloads.html"
-    regex(%r{href="https://releases.hashicorp.com/nomad/([0-9.]+)})
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
```

Comment:
I'm currently working on ensuring all livecheckables have a `url` in them, so it would be great if you can add the URL here too @miccal.

```suggestion
    url :head
    regex(/^v?(\d+(?:\.\d+)+)$/i)
```

Reply:
Thanks for your review @nandahkrishna.